pom.xml

```diff
@@ -4,14 +4,14 @@
   <!--
   export MAVEN_OPTS="$MAVEN_OPTS -Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n"
   mvn clean package -DskipTests jetty:run -Djetty.port=8082
+  mvn package -DskipTests jetty:run -Djetty.port=8082

```

Comment:
duplicate comment?

Reply:
No, one has a "clean" (which blasts the target folder, and therefore the database when building) and the other doesn't. It's a quick copy/paste efficiency when developing rather than typing them out